### DISCUSSION TITLE
Handling capitation with percent of billed

### DISCUSSION THREAD

**Discussion by PricedTransparently** [CMSgov/price-transparency-guide#474] [Q&A]
It is possible for a contract to have a pay method of "percent of billed", with a service type of "Capitated". In this scenario, we would have a negotiation_arrangement of "capitation" and a negotiated_type of "derived". The problem here is that our negotiated_rate would be a percentage value (not a dollar amount), but there is nothing in the schema that conveys this information. How should we be handling this scenario? Should we be making use of the additional_information object for this?

**Comment by shaselton-usds** [comment]
@PricedTransparently I just want to check that your understanding of "derived" falls inline with the [rule's definition.](https://www.federalregister.gov/d/2020-24591/p-1426)

It would seem that "derived" would be a dollar amount vs a percentage. 

You can certainly report a `negotiation_arrangement` as "capitation" and the `negotiation_type` as "percentage" if that is what you have. Also, note that [`negotiated_prices`](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-capitation-single-plan-sample.json#L31-L36) is an array to allow for multiple reporting of negotiation types -- so you may report a percentage as well as a derived amount if those are different.

**Reply by bryanbriegel** [reply]
The TiC rule calls for a dollar amount to be disclosed in the in-network negotiated rate files, and there is no allotment for reporting a percentage that I can see.  "the final rules clarify that plans and issuers are required to include for each item or service in the In-network Rate File, the base negotiated rate that applies before adjusting for participant, beneficiary, or enrollee -specific characteristics. The negotiated rate in the referenced-based pricing model must be represented as a dollar value that is the result of the calculation of the referenced amount and the applicable reference-based percentage" this aligns with the PRA packet, which also requires a dollar amount (not percentage) to be disclosed:  "the negotiated rate, reflected as a dollar amount, for each covered item or service under the plan or coverage that the plan or issuer has contractually agreed to pay an in-network provider,.."
